API/YAML design: Custom URL for MDM routes (#22547)
fleetdm/fleet: docs/Configuration/yaml-files.md

Can only be configured for all teams (`org_settings`).

##### client_url

> Update this URL if you're self-hosting Fleet and you want your hosts to talk to a different URL for MDM features (default: base URL of the Fleet instance provided during setup).

Reviewer: ```suggestion
Update this URL if you're self-hosting Fleet and you want your hosts to talk to a different URL for MDM features. (If not configured, hosts will use the base URL of the Fleet instance.)
```
